engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -278,4 +280,37 @@ public static synchronized void changeLoggersLevel(@Nonnull Map<String, String>
     }
     return ImmutableMap.copyOf(matchedClasses);
   }
+
+  /**
+   * Returns logger log file name
+   * @return logFileName
+   */
+  public static String getLogFileName()
+  {
+    Enumeration e = LogManager.getRootLogger().getAllAppenders();
+    while (e.hasMoreElements()) {
+      Appender appender = (Appender)e.nextElement();
+      if (appender instanceof FileAppender) {
+        return ((FileAppender)appender).getFile();
```

Comment:
I would say, let's keep it configurable for now? 
As there could be multiple file Appenders, it's not straightforward to figure out which one is being used for current exception. log4j can be configured to use different appenders for different classes. For the V1 of feature, we can keep it configurable and give default value as "RFA". We can further extend this feature to find out the appender for class logging the exception and then fetch the file name.